drucker_dashboard/models/evaluation_result.py

```diff
@@ -20,9 +21,23 @@ class EvaluationResult(db.Model):
     model_id = Column(Integer, nullable=False)
     data_path = Column(String(512), nullable=False)
     evaluation_id = Column(Integer, ForeignKey('evaluations.evaluation_id'), nullable=False)
-    result = Column(Text, nullable=False)
+    _result = Column(Text, nullable=False)
```

Comment:
Could you make an Issue about this field?
Storing json value is weird for me but I don't have a good idea :(